101-synapse-poc/azuredeploy.json

```diff
@@ -0,0 +1,621 @@
+                    "TIME_ZONE": { "value": "[parameters('TIME_ZONE')]" },
+                    "ResumeTime": { "value": "[parameters('ResumeTime')]" },
+                    "location": { "value": "[parameters('location')]" }
+                }
+            }
+        },
+        {
+            "apiVersion": "2019-06-01-preview",
+            "name": "[variables('workspaceName')]",
+            "location": "[parameters('location')]",
+            "type": "Microsoft.Synapse/workspaces",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "defaultDataLakeStorage": {
+                    "accountUrl": "[variables('defaultDataLakeStorageAccountUrl')]",
+                    "filesystem": "[variables('dlsFsName')]"
+                },
+                "sqlAdministratorLogin": "[parameters('sqlAdministratorLogin')]",
+                "sqlAdministratorLoginPassword": "[parameters('sqlAdministratorLoginPassword')]",
+                "managedVirtualNetwork": "default"
+            },
+            "resources": [
+                {
+                    "condition": "[equals(parameters('allowAllConnections'),'true')]",
+                    "apiVersion": "2019-06-01-preview",
+                    "dependsOn": [
+                        "[variables('workspaceName')]"
+                    ],
+                    "location": "[parameters('location')]",
+                    "name": "allowAll",
+                    "properties": {
+                        "startIpAddress": "0.0.0.0",
+                        "endIpAddress": "255.255.255.255"
+                    },
+                    "type": "firewallrules"
+                },
+                {
+                    "condition": true,
+                    "apiVersion": "2019-06-01-preview",
+                    "dependsOn": [
+                        "[variables('workspaceName')]"
+                    ],
+                    "location": "[parameters('location')]",
+                    "name": "AllowAllWindowsAzureIps",
+                    "properties": {
+                        "startIpAddress": "0.0.0.0",
+                        "endIpAddress": "0.0.0.0"
+                    },
+                    "type": "firewallrules"
+                },
+                {
+                    "apiVersion": "2019-06-01-preview",
+                    "dependsOn": [
+                        "[variables('workspaceName')]"
+                    ],
+                    "location": "[parameters('location')]",
+                    "name": "default",
+                    "properties": {
+                        "grantSqlControlToManagedIdentity": {
+                            "desiredState": "Enabled"
+                        }
+                    },
+                    "type": "managedIdentitySqlControlSettings"
+                }
+            ],
+            "dependsOn": [
```

Comment:
looks like a lot of the resources have properties in the wrong order, won't comment on them all, be sure to fix according to the BP guide